src/main/java/nl/tudelft/pixelperfect/gui/GameHeadsUpDisplay.java

```diff
@@ -59,6 +61,20 @@ public GameHeadsUpDisplay(AssetManager passedMan, Node passedGuiNode, int passed
    * Sets-up text display for the in-game HUD, along with all of its elements.
    */
   private void setupTextDisplay() {
+    // Set-up the bitmap text needed for HUD display.
+    setUpBitmapText();
+
+    // Attach the elements to the gui nodes.
+    guiNodes.attachChild(captainLog);
+    guiNodes.attachChild(shipHealth);
+    guiNodes.attachChild(teamScore);
+    guiNodes.attachChild(timeLeft);
```

Comment:
As I have outlined before, this causes nullpointers on refresh due to JMonkeyEngine's setup.  I have pushed the following commit as proof, and will then revert it.